src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+const { ExpenseServises } = require('../services/expenses.service');
+const { UserServices } = require('../services/users.service');
+const { STATUS_CODE } = require('../utils/constants');
+
+const getExpenses = (req, res) => {
+  const { userId, from, to, categories } = req.query;
+
+  let expenses = ExpenseServises.getExpenses();
+
+  if (userId) {
+    expenses = expenses.filter(expense => expense.userId === Number(userId));
+  }
+
+  if (categories) {
+    expenses = expenses.filter(expense =>
+      categories.includes(expense.category));
```

Comment:
if there are no `categories` in search params we will get `undefined`
if only 1 `categories` we will get just value (....?categories=1)
And if we have many categories we will get array of values (....?categories=1&categories=2&categories=3)

![image](https://github.com/mate-academy/node_accounting-app-v2/assets/86789161/ea010564-92a2-4116-9fcf-a66739d6d01f)

You code will crash if you have only 1 category